suite-common/wallet-core/src/blockchain/blockchainActions.ts

```diff
@@ -0,0 +1,56 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import { Network, NetworkSymbol } from '@suite-common/wallet-config';
```

Comment:
When you have added NetworkSymbol to imports, fix it everywhere in this file instead of Network['symbol'].

Reply:
https://github.com/trezor/trezor-suite/pull/6140/commits/5a38c9aea55e9811f7fecdf7679ca1a86081f8ec